tests/ecomm.tests.ts

```diff
@@ -40,6 +40,11 @@ describe('Ecomm tests', () => {
     scope = nock('https://api.duda.co')
   })
 
+  it('can list all products', async () => {
+    scope.get(`/api/sites/multiscreen/test_site/ecommerce/products`).reply(200, product)
+    return await duda.ecomm.products.list({ site_name: "test_site" })
```

Comment:
Can we add a couple tests that make sure the query params are sent along? Like offset, limit, etc.